flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -275,8 +276,9 @@
 	public static final ConfigOption<String> TASK_OFF_HEAP_MEMORY =
 		key("taskmanager.memory.task.off-heap.size")
 			.defaultValue("0b")
-			.withDescription("Task Heap Memory size for TaskExecutors. This is the size of off heap memory (JVM direct"
-				+ " memory or native memory) reserved for user code.");
+			.withDescription("Task Off-Heap Memory size for TaskExecutors. This is the size of off heap memory (JVM"
+				+ " direct memory and native memory) reserved for user code. The configured value will be fully counted"
```

Comment:
I think it should be `for tasks` rather than `for user code`, e.g. it can be used by Flink sql operators or state backends.

Reply:
Nice catch, thanks. Comment addressed.